### DISCUSSION TITLE
Unable to access WebUI

### DISCUSSION THREAD

**Discussion by earlbalai** [gravitational/teleport#9147] [General]
Hello all, I followed the instructions for Linux Standalone on the docs but when I try to access the WebUI it never connects.

**Comment by mburrows64** [comment]
Same .. Installed teleport-8.0.0-1 on RHEL8.4 ... there is no UI running on tcp/443

**Reply by mburrows64** [reply]
aah .. the webui needs to be installed separately. The quickstart guide doesn't mention that.

**Comment by mburrows64** [comment]
So if I run teleport configure with the --acme options it starts listening on tcp/443 .. but I don't want to use LetsEncrypt, I have an internal CA.
If I don't add the --acne options nothing is listening on tcp/443 

First question ... How do I use certs signed from internal CA?
Do I need to install the WebUI separately after installing teleport RPM? 
How do I do that?
Is there anther RPM or a different deployment method for UI? 
Is there a container I can pull that has everything I need and can run with podman?

Sorry for all the questions .. the doco isn't answering them. 

Thanks .. this looks like a cool solution, I'm keen to get it working for my use case.

**Reply by darius-fritsch** [reply]
I would recommend taking a look in the [reference section](https://goteleport.com/docs/setup/reference/config/) at the documentation.
